src/cascade/input_data/db/bundle.py

```diff
@@ -13,7 +13,16 @@
 CODELOG = logging.getLogger(__name__)
 
 # FIXME: There is a shared function that get's the official mapping, I think. Or an sql query at least.
-MEASURES = {6: "incidence", 9: "mtexcess", 5: "prevalence"}
+MEASURES = {
```

Comment:
The map to dismod names is currently inside the db.metadata as an Enum. I hope not to need it far from there.